content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst

```diff
@@ -0,0 +1,63 @@
+=================
+Add new equipment
+=================
+Using Odoo Maintenance, it is possible to track individual pieces of equipment, along with
+information about their maintenance requirements. To add a new piece of equipment, navigate to the
+:guilabel:`Maintenance` module, select :menuselection:`Equipments --> Machines & Tools --> Create`,
+and configure the equipment as follows:
+
+- :guilabel:`Equipment Name`: the product name of the piece of equipment
+- :guilabel:`Equipment Category`: the category that the equipment belongs to
+- :guilabel:`Company`: the company that owns the equipment
```

Comment:
```suggestion
- :guilabel:`Company`: the company that owns the equipment. Again this can be the company that's
  listed as the Odoo database owner, or it can be a vendor in the case of equipment rentals.
```